packages/elements/src/list/index.ts

```diff
@@ -253,7 +242,7 @@ export class List<T extends DataItem = ItemData> extends ControlElement {
    * @returns {void}
    */
   public first (): void {
-    const firstItem = this.itemMap.get(this.tabbableElements[0]);
+    const firstItem = this.itemMap.get(this.tabbableItems[0]);
```

Comment:
`tabbableItems !== tabbableElements` and includes all tababble nodes in the tree. This can return you the wrong information.

Reply:
Could you clarify a bit more ? It is going to be different anyway because in both cases `list` and `tree`, children won't be focusable.

Reply:
`tabbableElements` always returns the item element nodes (`ef-list > *`). `tabbableItems` goes through the whole tree I believe (including shadow root), so if you had a custom renderer with a focusable element in it, it would be included in the result.

Reply:
But like you said, this needs to change anyway now as the list items are no longer focuable. Custom renderers can render focusable items, however - I suppose that's down to the devs to manage though.